### PR TITLE
MISUV-3718 : Updating Claim a Refund page

### PR DESCRIPTION
**Pull request by AshokGitAccount**

### CODE REVIEW

File: app/views/CreditAndRefunds.scala.html [hunk 1]
```diff
@@ -43,170 +43,190 @@
...
-        @if(balance.get.availableCredit.get != 0){
-            <h2 class="govuk-heading-m">@{balance.get.availableCredit.get.toCurrency}
-                <span class ="govuk-body"> @messages("credit-and-refund.subHeading.has-credits")</span>
-            </h2>
-        } else {
-            <h2 class="govuk-heading-m">@{balance.get.firstPendingAmountRequested.get.toCurrency}
-                <span class ="govuk-body"> @messages("credit-and-refund.subHeading.has-no-credits")</span>
-            </h2>
-        }
+@if(balance.isDefined){
```
Comment (avelx) [inline app/views/CreditAndRefunds.scala.html:95]: if you can format code between lines: 96 - 106,

**Review by avelx — CHANGES_REQUESTED** [review]
minore change required: formatting

File: app/views/CreditAndRefunds.scala.html (hunk 1 above)
Comment (AshokGitAccount) [reply app/views/CreditAndRefunds.scala.html]: I reformatted whole file

**Review by andriikondiuk — APPROVED** [review]
Looks good to me

**Pull request merged**